### DISCUSSION TITLE
Meslo Nerd Font - Terminal Panel

### DISCUSSION THREAD

**Discussion by awamser** [zed-industries/zed#6665] [Feature Requests]
First off, the work you guys are doing is amazing.  I've been using Zed for a number of months and can't go back to my previous IDE.  My question is pretty low priority and not a huge issue.  I use OhMyZSH /w Powerlevel10k.  One of the requirements is Meslo Nerd Font; the icons are broken in the Zed Terminal panel.  

**From iTerm2**

<img width="867" alt="Screenshot 2024-01-08 at 5 04 37 PM" src="https://github.com/zed-industries/community/assets/318011/b3085215-2c0e-4547-8400-27f290631d5a">

**From Zed Terminal Panel**

<img width="1093" alt="Screenshot 2024-01-08 at 5 04 15 PM" src="https://github.com/zed-industries/community/assets/318011/f4b978bf-cd03-4d7d-b422-27111adb8603">

**Comment by tml** [comment]
You can change this by setting ```"terminal": {
    "font_family": "…"
  }``` in your `settings.json` file.

**Reply by awamser** [reply]
"terminal": { "font_family": "MesloLGS NF" }

**Reply by MaxStaszkiewicz** [reply]
or better yet, add it as a fallback font. The above suggestions will overwrite your currently selected font. 😁 

```bash
"terminal": { "font_fallbacks": ["MesloLGS NF"] }
```
